.buildkite/pipeline.yml

```diff
@@ -214,6 +214,7 @@ steps:
     command: |
       echo "--- :node: Downloading Binaries"
       buildkite-agent artifact download "*.zip" .
+      buildkite-agent artifact download "*.dmg" .
```

Comment:
Looks good. Should we add the same in the step that publishes dev builds?

Reply:
Ah, yes, good catch. I've added it in https://github.com/Automattic/studio/pull/333/commits/7b2c60875c4fd37821055c1c5e1cfb8f3235b65d.